### PR TITLE
Add Telegram bot config UI

### PR DESCRIPTION
**Pull request by athurg**
This PR is related #1740 .

It add UI support to config Telegram Bot.

- For `TelegramBotToken` of the whole Memos Server, you can login as Admin/Host user, navigate to *Admin-System* page, set your telegram bot token in `Telegram Bot Token` field.
- For `TelegramUserId` for each user of Memos, you can login as any user, navigate to *Basic-Preferences*, set your telegram userid in `Telegram User ID` field.

PS: You can get your telegram user id by send any message to your telegram bot after config the `TelegramBotToken` to your Memos Server.

### CODE REVIEW

File: web/src/components/Settings/PreferencesSection.tsx [hunk 1]
```diff
@@ -13,13 +15,17 @@ const PreferencesSection = () => {
...
   const { setting, localSetting } = userStore.state.user as User;
+  const [telegramUserId, setTelegramUserId] = useState<string>("");
   const visibilitySelectorItems = VISIBILITY_SELECTOR_ITEMS.map((item) => {
     return {
       value: item.value,
       text: t(`memo.visibility.${item.text.toLowerCase()}`),
     };
   });
 
+  useEffect(() => {
```
Comment (CorrectRoadH) [inline web/src/components/Settings/PreferencesSection.tsx:26]: Why need `useEffect`? 👀

Comment (athurg) [reply web/src/components/Settings/PreferencesSection.tsx:26]: The input value need to cache in storage, so we can submit it after all input done.

`useEffect` to load value from store while loading or other case .

Comment (CorrectRoadH) [reply web/src/components/Settings/PreferencesSection.tsx:26]: 🤔Can it be replaced like
```typescript
  useEffect(() => {
    globalStore.fetchSystemStatus();
  }, []);

  useEffect(() => {
    api.getSystemSetting().then(({ data: { data: systemSettings } }) => {
      const openAIConfigSetting = systemSettings.find((setting) => setting.name === "openai-config");
      if (openAIConfigSetting) {
        setOpenAIConfig(JSON.parse(openAIConfigSetting.value));
      }

      const telegramRobotSetting = systemSettings.find((setting) => setting.name === "telegram-robot-token");
      if (telegramRobotSetting) {
        setTelegramRobotToken(telegramRobotSetting.value);
      }
    });
  }, []);
```

**Review by CorrectRoadH — APPROVED** [review]
LGTM

File: web/src/components/Settings/SystemSection.tsx [hunk 2]
```diff
@@ -246,6 +270,26 @@ const SystemSection = () => {
...
+      <Input
+        className="w-full"
+        sx={{
+          fontFamily: "monospace",
+          fontSize: "14px",
+        }}
+        placeholder={t("setting.system-section.telegram-robot-token-placeholder")}
+        value={telegramRobotToken}
+        onChange={(event) => handleTelegramRobotTokenChanged(event.target.value)}
+      />
```
Comment (boojack) [inline web/src/components/Settings/SystemSection.tsx:292]:
```suggestion
      />
      <Divider className="!mt-3 !my-4" />
```

**Review by boojack — APPROVED** [review]

**Pull request merged**